src/openforms/scss/vendor/_react-select.scss

```diff
@@ -25,4 +25,5 @@
     100%,
     var(--of-admin-select-max-inline-size)
   );
+  max-inline-size: var(--of-admin-select-max-inline-size);
```

Comment:
I've had to do this in more places in other projects and now I no longer understand how `clamp` works

Reply:
Yeah i also dont fully understand why the max-inline-size was needed.. Maybe because a child element was growing, instead of the `.admin-react-select`. So that the max-inline-size prevent all growth?

I think the `clamp` just allows the parent to grow between a min and max value, but it doesn't work fully for flexbox child elements? That they don't respect the set value, and grow beyond? The `.admin-react-select` is used in a `flex-wrap` environment, which might also complicate things..

But I haven't checked/investigated it so deeply..